python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -165,59 +165,34 @@ def _preprocess_genetic_maps(map_directory_path: str) -> dict[int, list[int]]:
 
 
 def read_feather_in_chunks(file_path, columns=None, chunk_size=1000):
```

Comment:
```suggestion
def read_feather_in_chunks(file_path, columns=None, chunk_size=1000):
    """Read a Feather file in chunks as pandas DataFrames."""
    dataset = ds.dataset(file_path, format="feather")

    if columns:
        columns = list(columns)

    for batch in dataset.to_batches(batch_size=chunk_size, columns=columns):
        yield batch.to_pandas()
```

This may be more performant, because it will skip materializing the entire dataset in pandas before dropping columns.

Reply:
I think we'll want to set batch_readahead as well, 1 for now. IO will improve with larger batch_readahead, but at likely too much cost to memory based on our previous experiences with ancestry and saving